welcome/learn_more_about_openshift.adoc

```diff
@@ -30,44 +31,184 @@ Use the following sections to find content to help you learn about and use {prod
+
+|===
 
 [id="cluster-administrator"]
 == Cluster Administrator
 
-[options="header",cols="4*"]
+[options="header",cols="2*"]
 |===
-|Learn about {product-title} |Deploy {product-title} |Manage {product-title} |Additional resources
+|Learn about {product-title} cluster activities |Additional resources
 
-| link:https://www.openshift.com/blog/enterprise-kubernetes-with-openshift-part-one?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[Enterprise Kubernetes with OpenShift]
-| xref:../installing/overview/installing-preparing.adoc#installing-preparing[Installing {product-title}]
-| xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#using-insights-to-identify-issues-with-your-cluster[Using Insights to identify issues with your cluster]
-| xref:../support/getting-support.adoc#getting-support[Getting Support]
+| xref:../architecture/architecture.adoc#architecture-overview-architecture[Understand {product-title} management]
+| xref:../machine_management/index.adoc#machine-api-overview_overview-of-machine-management[Machine API] and xref:../architecture/control-plane.adoc#operators-overview_control-plane[Operators]
 
-| xref:../architecture/architecture.adoc#architecture[Architecture]
-| xref:../machine_configuration/index.adoc#machine-config-overview[Machine configuration overview]
-| xref:../observability/logging/cluster-logging.adoc#cluster-logging[Logging]
-| link:https://access.redhat.com/articles/4217411[OpenShift Knowledgebase articles]
+| xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]
+| xref:../installing/overview/cluster-capabilities.adoc#explanation_of_capabilities_cluster-capabilities[Optional cluster capabilities in {product-title {product-version}]
 
-| link:https://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[OpenShift Interactive Learning Portal]
-| xref:../networking/understanding-networking.adoc#understanding-networking[Networking]
-| xref:../observability/monitoring/monitoring-overview.adoc#monitoring-overview[Monitoring overview]
-| link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[{product-title} Life Cycle]
+|===
+
+[id="managing-changing-cluster-components"]
+=== Managing and changing cluster components
+
+[discrete]
+==== Managing cluster components
+
+[options="header",cols="2*"]
+|===
+|Learn about managing cluster components |Additional resources
+
+| Manage xref:../machine_management/index.adoc#machine-mgmt-intro-managing-compute_overview-of-machine-management[compute] and xref:../machine_management/index.adoc#machine-mgmt-intro-managing-control-plane_overview-of-machine-management[control plane] machines
+|xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[deploying health checks] and xref:../machine_management/applying-autoscaling.adoc#applying-autoscaling[applying autoscaling].
```

Comment:
These aren't really additional resource for the machine management content. The previous welcome page had:
> **Manage machines**: Manage compute and control plane machines in your cluster with machine sets, by deploying health checks, and applying autoscaling.

This meant:
Manage compute and control plane machines in your cluster 
- with machine sets,
- by deploying health checks, 
- and applying autoscaling

I would add two rows below the "Manage compute and control plane machines with machine sets" row so that there are three items as follows:

| Learn about managing cluster components |
| --- |
| Manage compute and control plane machines with machine sets |
| Deploy machine health checks |
| Apply autoscaling to an {product-title} cluster |

I can't think of any additional resources for any of these items at this time :)